### ISSUE TITLE
:fire: 6.0.0 iOS Firestore settings not applied

### ISSUE THREAD

**Issue by matb33** [invertase/react-native-firebase#2654]
## Issue

It appears any Firestore settings are not applied. The `setFirestoreSettings` method does not appear to do anything with the FIRFirestoreSettings object it creates.

The following may be all that is needed to fix this issue, but I leave this to you guys to judge:

```
diff --git a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCommon.m b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCommon.m
index 7aecefb..d045e67 100644
--- a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCommon.m
+++ b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCommon.m
@@ -77,6 +77,8 @@ + (void)setFirestoreSettings:(FIRFirestore *)firestore appName:(NSString *)appNa
   NSString *sslKey = [NSString stringWithFormat:@"%@_%@", FIRESTORE_SSL, appName];
   firestoreSettings.sslEnabled = (BOOL) [preferences getBooleanValue:sslKey defaultValue:firestore.settings.sslEnabled];
 
+  [FIRFirestore firestore].settings = firestoreSettings;
+    
   settingsLock[appName] = @(YES);
 
   return;
```

---

### iOS

<details><summary>Click To Expand</summary>
<p>

#### `ios/Podfile`:

- [ ] I'm not using Pods
- [ ] I'm using Pods and my Podfile looks like:

```ruby
# N/A
```

#### `AppDelegate.m`:

```objc
// N/A
```

</p>
</details>

---

### Android

<details><summary>Click To Expand</summary>
<p>

#### Have you converted to AndroidX?

- [ ] my application is an AndroidX application?
- [ ] I am using `android/gradle.settings` `jetifier=true` for Android compatibility?
- [ ] I am using the NPM package `jetifier` for react-native compatibility?

#### `android/build.gradle`:

```groovy
// N/A
```

#### `android/app/build.gradle`:

```groovy
// N/A
```

#### `android/settings.gradle`:

```groovy
// N/A
```

#### `MainApplication.java`:

```java
// N/A
```

#### `AndroidManifest.xml`:

```xml
<!-- N/A -->
```

</p>
</details>

---

## Environment

<details><summary>Click To Expand</summary>
<p>

**`react-native info` output:**

```
 OUTPUT GOES HERE
```

- **Platform that you're experiencing the issue on**:
  - [x] iOS
  - [ ] Android
  - [x] **iOS** but have not tested behavior on Android
  - [ ] **Android** but have not tested behavior on iOS
  - [ ] Both
- **`react-native-firebase` version you're using that has this issue:**
  - `6.0.0`
- **`Firebase` module(s) you're using that has the issue:**
  - `firestore`
- **Are you using `TypeScript`?**
  - `N`
  
</p>
</details>

---

Think `react-native-firebase` is great? Please consider supporting all of the project maintainers and contributors by donating via our [Open Collective](https://opencollective.com/react-native-firebase/donate) where all contributors can submit expenses. [[Learn More]](https://invertase.io/oss/react-native-firebase/contributing/donations-expenses)

- 👉 Check out [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter for updates on the library.

**Comment by Salakar** [comment]
cc @ehesp, one to fix next week, guess the tests didn't pick it up, maybe worth updating code to have a native method to read the settings just for testing?

**Comment by Salakar** [comment]
Or maybe an issue with how we set/read preferences

**Comment by Ehesp** [comment]
Yikes this is a bit silly! @matb33 feel free to PR that if you want. I'll get round to it later today or tomorrow if not.

**Comment by Salakar** [comment]
Hey 👋 this issue should now be fixed in v6.0.1, thanks. [[release notes]](https://invertase.io/oss/react-native-firebase/releases/v6.0.1)
